packages/common/src/discussions/_internal/DiscussionSchema.ts

```diff
@@ -13,10 +13,13 @@ export const DiscussionEditorTypes = ["hub:discussion:edit"] as const;
  * defines the JSON schema for a Discussion's editable fields
  */
 export const DiscussionSchema: IConfigurationSchema = {
-  required: ["name"],
+  required: ["title", "prompt"],
   type: "object",
   properties: {
-    name: ENTITY_NAME_SCHEMA,
+    title: ENTITY_NAME_SCHEMA,
```

Comment:
@juliannaeapicella despite this being labeled "Title" in the design mocks, I believe you'll still want this property to be "name" because we map the backing item's title (`item.title`) to `name` on the Hub entity (see [here](https://github.com/Esri/hub.js/blob/master/packages/common/src/core/_internal/getBasePropertyMap.ts#L57-L60)). In order for this to save properly from the `arcgis-hub-entity-editor`, the property name should be 1:1. You'll still be able to make the translated string "Title". This is what we do for projects as well